src/components/Home.tsx

```diff
@@ -1,21 +1,70 @@
-import { Button, Heading, Link } from "@chakra-ui/react";
+import {
+  Box,
+  Button,
+  Container,
+  HStack,
+  Heading,
+  Image,
+  Link,
+  Grid,
+  GridItem,
+  Center,
+  Text,
+  Card,
+  CardBody,
+  Flex
+} from "@chakra-ui/react";
 import { Link as RouterLink } from "react-router-dom";
+import { FaFacebookF, FaTwitter } from "react-icons/fa";
+import { BsInstagram } from "react-icons/bs";
+const params = {
+  iconSize: "30",
+  iconSpacing: "10px"
+};
```

Comment:
BLOCKING: Is this constant actually used anywhere?

Reply:
Right now it's not used anywhere but it was intended for social media icons